### PR TITLE
Rename Soroban RPC to Stellar RPC

### PR DESCRIPTION
**Pull request by 2opremio**

### CODE REVIEW

**Comment by 2opremio** [comment]
I still need to review the link renamings. I may have to revert a few of those changes

**Comment by briwylde08** [comment]
Hi! Is this PR doing the same as this one? https://github.com/stellar/stellar-docs/pull/1009

**Comment by 2opremio** [comment]
Ah, yes! I had missed the other PR. 

@aditya1702 should I close this PR and continue with #1009 ?

**Comment by 2opremio** [comment]
> When we come to renaming the paths, we need redirects from the old paths to the new paths.

I tried to keep the paths as they were. Where do you see renamings?

File: docs/data/rpc/admin-guide.mdx [hunk 1]
```diff
@@ -150,19 +150,19 @@ Now you can replace `--network standalone` with `--network futurenet` (or whatev
 We recommend the following ways to deploy your own RPC instance:
 
 1. Deploy to Kubernetes using [Helm](https://helm.sh/docs/intro/install/)
-2. Run the [soroban-rpc docker image](https://hub.docker.com/r/stellar/soroban-rpc) directly
+2. Run the [stellar-rpc docker image](https://hub.docker.com/r/stellar/soroban-rpc) directly
```
Comment (mollykarcher) [inline docs/data/rpc/admin-guide.mdx:153]: I think we should link to the stellar-rpc docker image

File: docs/networks/software-versions.mdx [hunk 2]
```diff
@@ -139,7 +139,7 @@ Note: Soroban CLI has been renamed to Stellar CLI.
 | Stellar Core | `v21.0.0` |
 | Soroban Rust SDK | `21.0.1-preview.3` |
 | Soroban CLI | `v21.0.0-rc.1` |
-| Soroban RPC | `v21.2.0` |
+| Stellar RPC | `v21.2.0` |
```
[the remark this answers is not in the document]
Comment (2opremio) [reply docs/networks/software-versions.mdx:142]: Makes sense

**Comment by urvisavla** [comment]
Not directly related to the PR but when I searched for all Stellar dashboards and couldn't find RPC one. Should we rename [soroban-rpc]( https://grafana.com/grafana/dashboards/19229-soroban-rpc/) dashboard?

File: docs/data/rpc/admin-guide.mdx [hunk 3]
```diff
@@ -194,7 +194,7 @@ helm install my-rpc stellar/soroban-rpc \
 
 This example of Helm chart usage highlights some key aspects:
 
-- Set the `global.image.sorobanRpc.tag` to a tag from the [soroban-rpc dockerhub repo](https://hub.docker.com/r/stellar/soroban-rpc) for the image version you want to run. Refer to [the software versions page](/docs/networks/software-versions) to find the correct tag for the Soroban release you are running.
+- Set the `global.image.sorobanRpc.tag` to a tag from the [stellar-rpc dockerhub repo](https://hub.docker.com/r/stellar/soroban-rpc) for the image version you want to run. Refer to [the software versions page](/docs/networks/software-versions) to find the correct tag for the Soroban release you are running.
```
Comment (mollykarcher) [inline docs/data/rpc/admin-guide.mdx:197]: You changed the tech here but not the dockerhub repo link

File: docs/data/rpc/admin-guide.mdx [hunk 4]
```diff
@@ -305,18 +305,18 @@ docker run -p 8001:8001 -p 8000:8000 \
...
 -v <CONFIG_FOLDER>:/config stellar/soroban-rpc \
 --config-path <RPC_CONFIG_FILENAME>
 ```
 
-You can use Soroban RPC itself to generate a starter configuration file:
+You can use Stellar RPC itself to generate a starter configuration file:
 
 ```bash
-docker run stellar/soroban-rpc gen-config-file > soroban-rpc-config.toml
+docker run stellar/soroban-rpc gen-config-file > stellar-rpc-config.toml
```
Comment (mollykarcher) [inline docs/data/rpc/admin-guide.mdx:319]: I would leave this one, because the source code in the repo isn't actually changed yet right?

**Comment by leighmcculloch** [comment]
> > When we come to renaming the paths, we need redirects from the old paths to the new paths.
> 
> I tried to keep the paths as they were. Where do you see renamings?

If we're going to keep the paths as is that works too. I saw your comments saying "I still need to review the link renamings" and I thought the paths were going to change too. If not, no worries.

**Review by mollykarcher — APPROVED** [review]

**Pull request merged**